source/includes/_imageannotation.mdown

```diff
@@ -874,3 +878,121 @@ The `response` field, which is part of the callback POST request and permanently
 <aside class="notice">
 See the <a href="#callbacks">Callback section</a> for more details about callbacks.
 </aside>
+
+## Layers
```

Comment:
Small comment here — you can actually use Slate to align the two so that the left-side is aligned with the right side

By the I mean, the "Lines" section is left of a "lines" example, "Polygon" section is left of a "polygons" example, etc. It'll make it easier to read I think

Reply:
Oh I think it's aligned already - in the screenshot above, the layers param example is to the left of the layers paragraph, and I think the way I did it is the same as it's done in the above sections. Is that right?

Reply:
Slate only lets you have this cool align feature for H2, not with H3 as I was using. So I had to put "Annotation Layers" as a separate section for that to work. Let me know if that seems right.

Reply:
great! I would put it above "Create Image Annotation Tasks" and call it "Image Annotation Layers"